src/Storefront/Resources/app/storefront/src/plugin/cookie/cookie-configuration.plugin.js

```diff
@@ -425,21 +427,63 @@ export default class CookieConfiguration extends Plugin {
         this.closeOffCanvas();
     }
 
-    acceptAllCookies() {
-        this.openOffCanvas(() => {
-            this._handleAcceptAll();
-        });
+    /**
+     * Accepts all cookies. Pass `true` to the loadIntoMemory parameter to load the DOM into memory instead of
+     * opening the OffCanvas menu.
+     *
+     * @param loadIntoMemory
+     */
+    acceptAllCookies(loadIntoMemory = false) {
+        if (loadIntoMemory) {
```

Comment:
I have unintentionally skipped this the last time - is this really necessary? I think firing the cookie update event would be enough no? Why is it necessary to parse an in-memory DOM?

Reply:
Normally it loads it into the offcanvas sidebar. Since we don't load the sidebar now, we need to do this virtually, in memory.